### ISSUE TITLE
A11y_code.visualstudio.com_APIGetStarted _YourFirstExtension_Audio-onlyandVideo-only(Pre-recorded) : Captions or text transcript is not provided for the videos

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2781]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://github.com/microsoft/vscode-docs/issues/2779](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
MAS Violated: MAS 1.2.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/api/get-started/your-first-extension](https://code.visualstudio.com/api/get-started/your-first-extension) OR
follow below steps:
1.Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "API" link.
3.From left navigation, select "Get Started".
4.Navigate to "Your First Extension" link.
5.Navigate to the first video present in the scenario.
6.Observe whether pre recorded captions and text transcript is present for the video

**Actual:**
Captions or text transcript is not provided for the videos.

This issue is observed in Edge, Chrome 

**Expected:**
For all the pre-recorded media content, captions or any transcript should be present so that people with hearing disability can understand what is explained in the video.​
​
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:​**
People with hearing disability will not be able to understand the information that is explained in the video  if captions or text transcript is not provided​

**MAS Reference:**
MAS 1.2.1 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ad33be23-f66c-4c3f-8e03-101b3e07fa4c}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ad33be23-f66c-4c3f-8e03-101b3e07fa4c})

**Attachment for Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b7972cfb-da09-4dba-8787-14d839ff4bbe}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b7972cfb-da09-4dba-8787-14d839ff4bbe})

**Comment by octref** [comment]
The video has no sound. Keys pressed are shown in the screen. The text above it provides full explanation.

People with hearing disabilities can read and fully understand the video.
